config/locales/client.en.yml

```diff
@@ -211,9 +211,10 @@ en:
       tools:
         back: "Back"
         short_title: "Tools"
-        new: "New Tool"
         name: "Name"
         name_help: "The unique name of the tool as used by the language model"
+        tool_name: "Tool Name"
+        tool_name_help: "Tool name, only include numbers, letters, and underscores"
```

Comment:
Let me know if you think there is a better copy for this tool_name_help and also the `name_help` copy as its value isn't unique actually.